[2022-12-11 19:22:31,251] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: etl_DAG.run_etl_process scheduled__2022-12-06T15:00:00+00:00 [queued]>
[2022-12-11 19:22:31,315] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: etl_DAG.run_etl_process scheduled__2022-12-06T15:00:00+00:00 [queued]>
[2022-12-11 19:22:31,317] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-12-11 19:22:31,322] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2022-12-11 19:22:31,324] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-12-11 19:22:31,401] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): run_etl_process> on 2022-12-06 15:00:00+00:00
[2022-12-11 19:22:31,431] {standard_task_runner.py:52} INFO - Started process 388 to run task
[2022-12-11 19:22:31,446] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'etl_DAG', 'run_etl_process', 'scheduled__2022-12-06T15:00:00+00:00', '--job-id', '228', '--raw', '--subdir', 'DAGS_FOLDER/etl_DAG.py', '--cfg-path', '/tmp/tmpq71sebll', '--error-file', '/tmp/tmpom_jzoin']
[2022-12-11 19:22:31,452] {standard_task_runner.py:80} INFO - Job 228: Subtask run_etl_process
[2022-12-11 19:22:31,837] {task_command.py:369} INFO - Running <TaskInstance: etl_DAG.run_etl_process scheduled__2022-12-06T15:00:00+00:00 [running]> on host 0163e0790208
[2022-12-11 19:22:32,321] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=***@example.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=etl_DAG
AIRFLOW_CTX_TASK_ID=run_etl_process
AIRFLOW_CTX_EXECUTION_DATE=2022-12-06T15:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-12-06T15:00:00+00:00
[2022-12-11 19:22:39,913] {python.py:173} INFO - Done. Returned value was: None
[2022-12-11 19:22:40,116] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=etl_DAG, task_id=run_etl_process, execution_date=20221206T150000, start_date=20221211T192231, end_date=20221211T192240
[2022-12-11 19:22:40,231] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-12-11 19:22:40,399] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
